nss-cohort-40/c40-keahua-arboretum-da_beach_combers
# Adi holding

# Description
Added to the report the ability to show animals in river
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Check into branch adi-holding and run the program, add a river, then a dolphin to the river and then display the report
# Checklist:
